platform/ios/src/MLNMapView.mm

```diff
@@ -7501,4 +7502,15 @@ - (void)setShowsHeading:(BOOL)showsHeading
     self.showsUserHeadingIndicator = showsHeading;
 }
 
+// MARK: Tile Cache
+
+- (void)setTileCacheEnabled:(BOOL)enable {
```

Comment:
@alexcristici Should we just prefix this with `_experimental_`?

Reply:
Yes, I think is the a good way to make sure everybody is aware.